codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexCodecTest.java

```diff
@@ -286,7 +286,7 @@ public void idleOutboundStreamShouldNotWriteResetFrameOnClose() {
     }
 
     @Test
-    public void outboundStreamShouldWriteResetFrameOnClose_headersSent() {
+    public void outboundStreamShouldWriteResetFrameOnClose_headersSent() throws Exception {
```

Comment:
`throws Exception` can be removed here.

Reply:
Did we want a unit test in `Http2MultiplexCodecTest` that validates we don't try to write a reset under the conditions in question?

Reply:
done.